lib/assets/javascripts/cartodb3/components/infobox/infobox-item-view.js

```
@@ -33,12 +33,14 @@ module.exports = CoreView.extend({
       this._mainLabel = opts.mainAction.label;
       this._mainType = opts.mainAction.type;
       this._mainDisabled = opts.mainAction.disabled;
+      this._mainPosition = opts.mainAction.position;
```

if not optional, better to be explicit and throw an error if param is not present.